pytorch_lightning/utilities/auto_restart.py

```diff
@@ -571,3 +571,18 @@ def reload_dataloader_state_dict(dataloader: DataLoader, state_dict: Dict[str, A
 
     else:
         raise MisconfigurationException("This shouldn't happen. Please, open an issue on PyTorch Lightning Github.")
+
+
+def _detect_fault_tolerant_training_mode() -> _FaultTolerantTrainingMode:
+    """This utility detects if Fault Tolerant is activated and maps its value to `_FaultTolerantTrainingMode`."""
+    env_value = os.getenv("PL_FAULT_TOLERANT_TRAINING", "0")
+    if env_value == "0":
+        return _FaultTolerantTrainingMode.DISABLED
+    elif env_value == "1":
```

Comment:
Should we do something like

```suggestion
    if env_value in ("0", "disabled"):  # 0 kept for compatibility
        return _FaultTolerantTrainingMode.DISABLED
    elif env_value == "automatic":
```

Reply:
Done !